ext/opentelemetry-ext-flask/README.rst

```diff
@@ -16,6 +16,16 @@ Installation
 
     pip install opentelemetry-ext-flask
 
+Configuration
+-------------
+
+Exclude lists
+*************
+Excludes certain hosts and paths from being tracked. Pass in comma delimited string into environment variables.
+
+Excluded hosts: OPENTELEMETRY_PYTHON_FLASK_EXCLUDED_HOSTS
```

Comment:
It needs to be explained here that the values of these environment variables are regexes that should match with everything but the `r"(https?|ftp)://.*?/"` (are there any other protocols besides `http`, `https` and `ftp` that we want to take into consideration?) part of the URL the user wants excluded from tracing.

Reply:
I've included an explanation of what hosts and paths means. I don't think we need to inform the user of how the implementation is actually doing this. As for other protocols, these are the ones being excluded in OpenCensus, any new protocols I feel can be added later on.